hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -97,10 +104,32 @@ public class DBScanner implements Callable<Void>, SubcommandWithParent {
 
   private List<Object> scannedObjects;
 
+  public static byte[] getValueObject(
+      DBColumnFamilyDefinition dbColumnFamilyDefinition) throws IOException {
+    String str = dbColumnFamilyDefinition.getKeyType().getName();
```

Comment:
Its better to use Class instead of String to handle cases in which certain classes moved to different package.
```suggestion
     Class<?> keyType = dbColumnFamilyDefinition.getKeyType();
    if (keyType.equals(String.class)) {
      return startKey.getBytes();
    } else if (keyType.equals(ContainerID.class)) {
      return new ContainerID(Long.parseLong(startKey)).getBytes();
    } else if (keyType.equals(Long.class)) {
      return Longs.toByteArray(Long.parseLong(startKey));
    } else if (keyType.equals(PipelineID.class)) {
      return PipelineID.valueOf(UUID.fromString(startKey)).getProtobuf()
          .toByteArray();
```

Reply:
Done